Terminal.Gui/Core/Event.cs

```diff
@@ -302,6 +332,11 @@ public class KeyEvent {
 		/// </summary>
 		public Key Key;
 
+		/// <summary>
+		/// Check if the Shift key was pressed or not.
```

Comment:
I suggest:

    /// Indicates the "shift" state of the various shift-keys (e.g. Shift, Alt, Ctrl, NumLock, ...).